include/CppUTest/CppUTestConfig.h

```diff
@@ -61,34 +61,33 @@
 
 /* Do we use Standard C or not? When doing Kernel development, standard C usage is out. */
 #ifndef CPPUTEST_USE_STD_C_LIB
- #ifdef CPPUTEST_STD_C_LIB_DISABLED
-  #define CPPUTEST_USE_STD_C_LIB 0
- #else
-  #define CPPUTEST_USE_STD_C_LIB 1
- #endif
+#ifdef CPPUTEST_STD_C_LIB_DISABLED
+#define CPPUTEST_USE_STD_C_LIB 0
+#else
+#define CPPUTEST_USE_STD_C_LIB 1
+#endif
 #endif
-
 
 /* Do we use Standard C++ or not? */
 #ifndef CPPUTEST_USE_STD_CPP_LIB
- #ifdef CPPUTEST_STD_CPP_LIB_DISABLED
-  #define CPPUTEST_USE_STD_CPP_LIB 0
- #else
-  #define CPPUTEST_USE_STD_CPP_LIB 1
- #endif
+#ifdef CPPUTEST_STD_CPP_LIB_DISABLED
+#define CPPUTEST_USE_STD_CPP_LIB 0
+#else
+#define CPPUTEST_USE_STD_CPP_LIB 1
+#endif
 #endif
```

Comment:
I don't like these changes, this makes it less readable.

Reply:
Agreed. I'll set `IndentPPDirectives: AfterHash` to follow the existing style.